ipam/mas.go

```diff
@@ -5,129 +5,193 @@ package ipam
 
 import (
 	"encoding/json"
+	"io/ioutil"
 	"net"
-	"net/http"
-	"time"
+	"runtime"
+	"strings"
 
-	"github.com/Azure/azure-container-networking/common"
 	"github.com/Azure/azure-container-networking/log"
 )
 
 const (
-	// Host URL to query.
-	masQueryUrl = "http://169.254.169.254:6642/ListNetwork"
-
-	// Minimum time interval between consecutive queries.
-	masQueryInterval = 10 * time.Second
+	defaultLinuxFilePath = "/etc/kubernetes/interfaces.json"
+	defaultWindowsFilePath = `c:\k\interfaces.json`
```

Comment:
just a small question -> does this json file exists after reboot?

Reply:
Yes